Week-3/assignment-3.4/input.js

```diff
@@ -0,0 +1,18 @@
+function createStack() {
+    return {
+        items:[],
+        push(item) {
+            this.items.push(item);
```

Comment:
instead of using built in push and pop functions please try to keep track of stack top and return using that.